src/python_testing/TC_TestEventTrigger.py

```diff
@@ -110,6 +112,104 @@ async def test_correct_key_invalid_code(self):
                 )
             )
 
+    @async_test_body
+    async def test_multiple_command_request_feature_present_if_needed(self):
+        dev_ctrl = self.default_controller
+        test_event_triggers_enabled = await self.read_single_attribute(
+            dev_ctrl,
+            self.dut_node_id,
+            endpoint=0,
+            attribute=Clusters.GeneralDiagnostics.Attributes.TestEventTriggersEnabled
+        )
+        asserts.assert_true(test_event_triggers_enabled, "This test expects Test Event Triggers are Enabled")
+
+        max_paths_per_invoke = await self.read_single_attribute(self.default_controller, node_id=self.dut_node_id, endpoint=0, attribute=Clusters.BasicInformation.Attributes.MaxPathsPerInvoke)
+        feature_map = await self.read_single_attribute(self.default_controller, node_id=self.dut_node_id, endpoint=0, attribute=Clusters.GeneralDiagnostics.Attributes.FeatureMap)
+        accepted_commands = await self.read_single_attribute(self.default_controller, node_id=self.dut_node_id, endpoint=0, attribute=Clusters.GeneralDiagnostics.Attributes.AcceptedCommandList)
+        generated_commands = await self.read_single_attribute(self.default_controller, node_id=self.dut_node_id, endpoint=0, attribute=Clusters.GeneralDiagnostics.Attributes.GeneratedCommandList)
+
+        has_multiple_paths_per_invoke = max_paths_per_invoke > 1
+        has_data_model_test_feature = (feature_map & Clusters.GeneralDiagnostics.Bitmaps.Feature.kDataModelTest) != 0
+        has_payload_test_request_command = (Clusters.GeneralDiagnostics.Commands.PayloadTestRequest.command_id in accepted_commands)
+        has_payload_test_response_command = (
+            Clusters.GeneralDiagnostics.Commands.PayloadTestResponse.command_id in generated_commands)
+
+        logging.info(f"Number of supported invoke paths reported: {max_paths_per_invoke}")
+        if has_multiple_paths_per_invoke:
+            asserts.assert_true(has_data_model_test_feature, "Must have DMTEST feature when > 1 invoke paths supported")
```

Comment:
Is this CI only or is this required in the spec / test plan somewhere (not a blocker for this PR - this condition is correct for CI. Just wondering if this needs to be enforced at cert)

Reply:
This will come in a separate test plan update.